src/redux/books/books.js

```diff
@@ -0,0 +1,74 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [{
```

Comment:
### Heads-Up (Previous CR might leave out the comment because your app didn't run at all.)

Let's set the initial state to an empty array as mentioned in the project instructions.

![image](https://user-images.githubusercontent.com/31897434/130609331-52c5cb11-9197-47e6-8616-88f3da7528a4.png)

Reply:
I initialized the state to an empty array